apps/mobile/src/components/sheets/warning-sheet.layout.tsx

```diff
@@ -59,9 +58,6 @@ export function WarningSheetLayout({
                   />
                 </Box>
               }
-              onPressSupport={() => {
```

Comment:
I got rid of this as we are just showing a `?` in `Remove wallet` with no link. 

We give the user a lot of information there so I'm not sure we need the link but will double check cc @markmhendrickson 

![Screenshot 2024-12-09 at 10 20 57](https://github.com/user-attachments/assets/57dd3ad7-0184-4c57-8d8a-cf061ab300a9)

Reply:
I have added a placeholder URL here [as per](https://trustmachines.slack.com/archives/C05LRS7G44R/p1733740921786479?thread_ts=1733381141.338449&cid=C05LRS7G44R)

Reply:
I'd recommend we keep the link but cut down on the text. We can do this later via Crowdin though, so it's not imperative for this week's release